src/it/resources/nodes.conf

```diff
@@ -0,0 +1,81 @@
+nodes = [{
+  waves {
```

Comment:
Isn't this too much to be consistent? 
I mean wallet.seed produces priv+pub key, pub-key produces address.
We can easily get to incosistent state if we make things non-algebraic.

As (I guess) we need some of the params just for the reference, we can make them comments.